ui/components/multichain/account-details/account-details.js

```diff
@@ -82,6 +81,7 @@ export const AccountDetails = ({ address }) => {
           />
         </PopoverHeader>
       }
+      onClose={onClose}
```

Comment:
Oh wow, this is a good thing to know. 

@garrettbear @georgewrmarshall fyi when using the PopoverHeader `onClose` event within a Popover `title` prop, the event for the Popover closing "onBlur" does not trigger